packages/graphql/tests/integration/interface-relationships/read.int.test.ts

```diff
@@ -125,7 +125,7 @@ describe("interface relationships", () => {
 
             expect(gqlResult.errors).toBeFalsy();
 
-            expect(gqlResult.data?.actors[0].actedIn).toHaveLength(2);
+            expect((gqlResult.data as any)?.actors[0].actedIn).toHaveLength(2);
```

Comment:
I see a lot of these changes in the tests, anyway we can improve this so tests are not filled with these?.

maybe a test utility function that takes care of that when running graphql? something akin to this:
```ts
function runGraphql(options) {
      return graphql(options) as Promise<{errors?: Error, data: any}>
}

```

Reply:
I agree that some utility like this would be useful though, It may not be necessary to add it to this PR.